PendingReleaseNotes.md

```diff
@@ -2,19 +2,24 @@
 
 ## Breaking Changes
 
-* The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. See #9550
-* In the helm charts, all Ceph components now have default values for the pod resources. The values can be modified or removed in values.yaml depending on cluster requirements.
-* Prometheus rules are installed by the helm chart. If you were relying on the cephcluster setting `monitoring.enabled` to create the prometheus rules, they instead need to be enabled by setting `monitoring.createPrometheusRules` in the helm chart values.
-* The `region` field for OBC Storage class is ignored, the RGW server always works with s3 client using `us-east-1` as region.
- 
+* The MDS liveness and startup probes are now configured by the CephFilesystem CR instead of the
+  CephCluster CR. To apply the MDS probes, they need to be specified in the CephFilesystem CR. See the
+  [CephFilesystem doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. See #9550
+* In the Helm charts, all Ceph components now have default values for the pod resources. The values
+  can be modified or removed in values.yaml depending on cluster requirements.
+* Prometheus rules are installed by the Helm chart. If you were relying on the cephcluster setting
```

Comment:
Generally I just reformatted this to keep verb tense, use capital letters as appropriate, and use the longer/clearer names of things here.